plugin/src/main/java/io/deephaven/plugin/type/ObjectTypeClassBase.java

```diff
@@ -13,6 +13,31 @@
  * @param <T> the class type
  */
 public abstract class ObjectTypeClassBase<T> extends ObjectTypeBase {
+    public abstract static class FetchOnly<T> extends ObjectTypeClassBase<T> {
+
+        public FetchOnly(String name, Class<T> clazz) {
+            super(name, clazz);
+        }
+
+        @Override
+        public final MessageStream clientConnectionImpl(T object, MessageStream connection) {
+            StreamExporterImpl exporter = new StreamExporterImpl();
+
+            try {
+                writeToImpl(exporter, object, exporter.outputStream());
+            } catch (IOException e) {
+                e.printStackTrace();
```

Comment:
needs better error handling, possibly should share impl with its superclass (sort of).